library/webpack-loaders/web-worker-client-loader.js

```diff
@@ -0,0 +1,70 @@
+const { relative } = require('path')
+
+module.exports = WebworkerClientLoader
+
+function WebworkerClientLoader(source, map) {
+  const filename = relative(this.rootContext, this.resourcePath)
+  return createClientCode({ filename })
+}
+
+function createClientCode({ filename }) {
```

Comment:
Wat is mis ik als ik denk dat alleen een `ref` genoeg zou moeten zijn? Deze code is een stuk complexer geworden dan dat ik in eerste instantie dacht. Weet niet of we dat moeten willen want je hebt best wel wat kennis nodig om dit nu goed te kunnen lezen.

Het heeft wel een groot voordeel dat je nu makkelijker tegen de webworker kunt praten en je daar eigenlijk geen setup meer hebt. Dit maakt het wel heel gebruiksvriendelijk voor mensen die hier iets mee willen doen.